lib/uwb/include/uwb/UwbSession.hxx

```diff
@@ -136,12 +136,21 @@ public:
     SetSessionStatus(const uwb::protocol::fira::UwbSessionStatus& status);
 
     /**
-     * @brief Get the application configuration parameters for this session.
+     * @brief Get the Application Configuration Parameters object
      *
+     * @param requestedTypes leave this as an empty vector to request all parameters
      * @return std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameter>
      */
     std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameter>
-    GetApplicationConfigurationParameters();
+    GetApplicationConfigurationParameters(std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameterType> requestedTypes);
+
+    /**
+     * @brief Set the Application Configuration Parameters object
+     *
+     * @param params
```

Comment:
This comment needs to be updated

Reply:
ok, I think I _finally_ got all of them now 😭, let me know if you spot any that are missing!